From 9be108136dc5e7ef0143a3ae4338476ffa6377e9 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Thu, 27 Feb 2020 21:13:19 +0100
Subject: [PATCH] tests: Move memory_monitor_tests under
 installed_tests_enabled

So they're not installed when the latter is disabled.
---
 gio/tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/tests/meson.build b/gio/tests/meson.build
index 695ad1f8f..76e28fc76 100644
--- a/gio/tests/meson.build
+++ b/gio/tests/meson.build
@@ -521,32 +521,32 @@ if installed_tests_enabled
   )
   install_subdir('static-link', install_dir : installed_tests_execdir)
   install_data('static-link.py', install_dir : installed_tests_execdir)
-endif
 
 memory_monitor_tests = [
   'memory-monitor-dbus',
   'memory-monitor-portal',
 ]
 
 foreach memory_monitor_test : memory_monitor_tests
   cdata = configuration_data()
   cdata.set('installed_tests_dir', installed_tests_execdir)
   cdata.set('program', memory_monitor_test + '.py')
   configure_file(
     input: installed_tests_template,
     output: memory_monitor_test + '.test',
     install_dir: installed_tests_metadir,
     configuration: cdata
   )
   cdata = configuration_data()
   cdata.set('libexecdir', join_paths(glib_prefix, get_option('libexecdir')))
   configure_file(
     input: memory_monitor_test + '.py.in',
     output: memory_monitor_test + '.py',
     install_dir : installed_tests_execdir,
     configuration: cdata,
   )
 endforeach
+endif
 
 if not meson.is_cross_build() or meson.has_exe_wrapper()
 
-- 
2.25.1

